tools/Docker-dev/Dockerfile

```diff
@@ -0,0 +1,143 @@
+# 
+# Dockerfile for Scipy development 
+# 
+# Usage: 
+# -------
+# 
+# To make a local build of the container, from the 'Docker-dev' directory:
```

Comment:
minor: Directory and file names exclusively use underscores, and all-lowercase would also be easier. Could you change to `docker_dev`.